src/_content/posts/introducing-keynote-mario-munoz.md

```diff
@@ -0,0 +1,59 @@
+
+## Can you give us a teaser about what your talk is about?
+
+>I am very interested in the intersection of people and technology. What role does community have in our technological journey? How do we see ourselves within the context of a "technical" community?  At a recent conference (North Bay Python), I spoke about building bridges between communities. My talk at DjangoCon will be an extension of this. My current working title is "How to be a developer, and other lies we tell ourselves." We'll be going through a journey that brings us in and out of different mindsets as developers, and what role community plays in all of this. As many of my talks, I often have many biographical/personal/meta examples...
+
+## What are you most looking forward to at DjangoCon US 2024?
+
+>I am most excited about engaging with the community. I am currently involved in the Djangonaut program and look forward to seeing fellow Djangonauts there. I've also seen that the talks have been posted, and some of them look fantastic. I'm interested in all things hypermedia, so definitely will be catching those htmx-related talks.
+
+## What are your favorite topics at conferences?
+
+>As mentioned, I'm excited about the hypermedia. But I always like hearing about broad topics, such as the past, present, and future of Django talk by Will, or Jacob's talk on what the DSF could accomplish with 4x the budget.
+
+## How do you view the connections made at conferences?
+
+> The connections we make are invaluable. As an introvert, like many of us, I seek meaningful and strong connections. The Django community, in particular, has been a welcoming space for these interactions.
+
+## Can you tell us about your career journey?
+
+> I started my career in payroll and then moved to an HR analyst role, providing IT-related support. It was through networking that I met people who helped me transition into engineering. I met some key contacts at PyCon.
+
+## How did you approach your career growth?
+
+> I purposely leaned into areas where I wasn’t as strong, getting out there and trying to speak. It’s a long-term investment, and it takes time to get there. Purposefully plugging into the community and focusing on what you can do with code, whether you're a newcomer or experienced, is crucial. Learning not to gatekeep is also important.
+
+## When did you start speaking at conferences, and what has your experience been like?
+
+> I started speaking in 2022, giving talks at PyCon and PyOhio. I've done tutorials and talks, and it's been an incredible experience that has helped me grow both personally and professionally.
+
+## What do you like to do outside of your normal work day?
+
+>Beach days with the family are always fun, or adventuring in a new/unknown hiking area. And, well, I have a blog called Python By Night, so that kind of speaks for itself.
```

Comment:
```suggestion
>Beach days with the family are always fun, or adventuring in a new/unknown hiking area. And, well, I have a blog called [Python By Night](https://pythonbynight.com/), so that kind of speaks for itself.
```